pages/styles/common.css

```diff
@@ -61,4 +61,16 @@ h6{
 
 a{
     text-decoration: none;
+}
+
+.header{
+    padding: 1rem 4rem;
+}
+
+.header__nav-right{
+    margin-left: 0px;
+}
```

Comment:
is this mandatory?

Reply:
Yes actually i am using the header from component library i build in that margin-left is auto but here i don't want that.